### PR TITLE
docs: Theme palette variables update

### PR DESCRIPTION
**Pull request by starryeyez024**
## Documentation

Update docs to reflect new variables

### CODE REVIEW

File: docs/content/theme/theme-palette.md [hunk 1]
```diff
@@ -27,101 +27,169 @@ These variables are global hooks to override colors, fonts, spacing and more thr
...
-| `--pfe-theme--link-decoration--on-dark` | Text decoration | none |
-| `--pfe-theme--link-decoration--hover--on-dark` | Text decoration | underline |
-| `--pfe-theme--link-decoration--focus--on-dark` | Text decoration | underline |
-| `--pfe-theme--link-decoration--visited--on-dark` | Text decoration | none |
-| `--pfe-theme--link-decoration--on-saturated` | Text decoration | underline |
-| `--pfe-theme--link-decoration--hover--on-saturated` | Text decoration | underline |
-| `--pfe-theme--link-decoration--focus--on-saturated` | Text decoration | underline |
-| `--pfe-theme--link-decoration--visited--on-saturated` | Text decoration | underline | 
\ No newline at end of file
+### Color variable
```
Comment (starryeyez024) [inline docs/content/theme/theme-palette.md:30]:
```suggestion
### Color variables 
```

File: docs/content/theme/theme-palette.md [hunk 2]
```diff
@@ -27,101 +27,169 @@ These variables are global hooks to override colors, fonts, spacing and more thr
...
+| `--pfe-theme--grid-breakpoint--lg` | 992px |
+| `--pfe-theme--grid-breakpoint--xl` | 1200px |
+| `--pfe-theme--box-shadow--sm` | 0 0.0625rem 0.125rem 0 rgba(3, 3, 3, 0.2) |
+| `--pfe-theme--box-shadow--md` | 0 0.125rem 0.0625rem 0.0625rem rgba(3, 3, 3, 0.12), 0 0.25rem 0.6875rem 0.375rem rgba(3, 3, 3, 0.05) |
+| `--pfe-theme--box-shadow--lg` | 0 0.1875rem 0.4375rem 0.1875rem rgba(3, 3, 3, 0.13), 0 0.6875rem 1.5rem 1rem rgba(3, 3, 3, 0.12) |
+| `--pfe-theme--box-shadow--inset` | inset 0 0 0.625rem 0 #fafafa |
+| `--pfe-theme--animation-speed` | 0.3s |
+| `--pfe-theme--animation-timing` | cubic-bezier(0.465, 0.183, 0.153, 0.946) |
+| `--pfe-theme--opacity` | 0.09 |
+| `--pfe-theme--modal` | 99 |
```
Comment (starryeyez024) [inline docs/content/theme/theme-palette.md:192]: @castastrophe should these last 4 variables have z-index in the name? in the redhat-theme/theme.css we have:

```
  --pfe-theme--zindex--modal: 1001;
  --pfe-theme--zindex--navigation: 103;
  --pfe-theme--zindex--subnavigation: 95;
  --pfe-theme--zindex--content: 0;
```

Comment (castastrophe) [reply docs/content/theme/theme-palette.md:192]: Yes, they should

Comment (starryeyez024) [reply docs/content/theme/theme-palette.md:192]: opened a separate issue for this: https://github.com/patternfly/patternfly-elements/issues/1034

**Pull request merged**